### ISSUE TITLE
Incorrect rendered formula for semiring

### ISSUE THREAD

**Issue by albert-github** [libsemigroups/libsemigroups#202]
I'm not sure how the API documentation is generated, but seen the comments and commands it looks like it is done by doxygen (or something alike).

In the file `https://github.com/libsemigroups/libsemigroups/blob/master/include/libsemigroups/semiring.hpp` we have around line 41 the following comment:
```
  //! * Mulitplication is left and right distributive over addition, i.e.
  //! \f$a \times (b + c) = a \times b + a \times c\f$ and \f$(a + b)
  //! \times c = (a \times c) + (b \times c)\f$ for all \f$a,b,c\in R\f$;
```
when using doxygen on this (see e.g. https://fossies.org/dox/gap-4.11.0/classlibsemigroups_1_1Semiring.html Detailed Description. The Gap package uses this include file and here I saw the problem) a bit a strange formula (with `//!` in it) appears and later on some code appears that should be text.
The cause of this is the comment conversion from `//!` to `/**` comments by doxygen and the fact that an inline formula (`\f$`) is split over multiple lines and code in an inline formula is seen as is and not manipulated so the `//!` remains (Note in the mean time this problem has been fixed in doxygen as well, https://github.com/doxygen/doxygen/pull/7726, but I think it is also better to fix it in libsemigroups as well)/
The best / easiest solution is just to keep the inline formula on one line:
```
  //! * Mulitplication is left and right distributive over addition, i.e.
  //! \f$a \times (b + c) = a \times b + a \times c\f$ and 
  //! \f$(a + b) \times c = (a \times c) + (b \times c)\f$ for all \f$a,b,c\in R\f$;
```

**Comment by james-d-mitchell** [comment]
@albert-github thanks for the report, this is very easy to fix, I'll be happy to fix it in the next release.

**Comment by slel** [comment]
Take the opportunity to change "Mulitplication" -> "Multiplication".

**Comment by james-d-mitchell** [comment]
Resolved in v1.1.0
